app/src/main/kotlin/io/imhungry/model/TagEntity.kt

```diff
@@ -0,0 +1,10 @@
+package io.imhungry.model
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "tag")
+data class TagEntity(
+    @PrimaryKey @ColumnInfo(name = "tag") val tag: String
+)
```

Comment:
Again, you should have an auto-generated ID column with tag having a unique constraint.